src/test/java/step1/AdditionTest.java

```diff
@@ -0,0 +1,24 @@
+package step1;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.math.BigDecimal;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import step1.domain.Calculator;
+import step1.domain.Number;
+
+public class AdditionTest {
+    @Test
+    @DisplayName("덧셈 연산")
+    public void additionTest() {
+        String operator = "+";
+        Number left = Number.of(1);
+        Number right = Number.of(2);
+        Calculator calculator = new Calculator();
+        Number result = calculator.calculate(left, right, operator);
+        assertThat(result.getValue()).isEqualTo(BigDecimal.valueOf(3));
```

Comment:
```suggestion
                assertThat(result).isEqualTo(Number.of(3));
```
값을 꺼내서 비교하기보단 객체 자체를 비교하는건 어떻까요?